packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -4,12 +4,20 @@
  * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
+/* istanbul ignore next */
```

Comment:
Sorry, maybe it needs to be the first line of the file, before the license? I haven't tried this locally yet.